### PR TITLE
514 - Dropwizards implementation

### PR DESCRIPTION
**Pull request by jdesiloniz**
Solves #514 

This PR adds support for Dropwizard metrics for both clients and servers.

### CODE REVIEW

File: modules/internal/src/main/scala/higherkindness/mu/rpc/internal/metrics/MetricsOps.scala [hunk 1]
```diff
@@ -40,3 +40,40 @@ trait MetricsOps[F[_]] {
       classifier: Option[String]): F[Unit]
 
 }
+
+object MetricsOps {
+
+  sealed trait GrpcStatus
```
Comment (fedefernandez) [inline modules/internal/src/main/scala/higherkindness/mu/rpc/internal/metrics/MetricsOps.scala]:
```suggestion
  sealed trait GrpcStatus extends Product with Serialization
```

**Comment by jdesiloniz** [comment]
@fedefernandez Thanks! Addressed :)

File: modules/internal/src/main/scala/higherkindness/mu/rpc/internal/metrics/MetricsOps.scala [hunk 2]
```diff
@@ -40,3 +40,40 @@ trait MetricsOps[F[_]] {
       classifier: Option[String]): F[Unit]
 
 }
+
+object MetricsOps {
+
+  sealed trait GrpcStatus extends Product with Serialization
```
Comment (fedefernandez) [inline modules/internal/src/main/scala/higherkindness/mu/rpc/internal/metrics/MetricsOps.scala]:
```suggestion
  sealed trait GrpcStatus extends Product with Serializable
```

**Comment by jdesiloniz** [comment]
@fedefernandez @juanpedromoreno addressed both comments! Thanks!

**Pull request merged**